app/setting/myprofile/page.tsx

```diff
@@ -57,15 +154,23 @@ const MyProfile = () => {
           </Head>
           <ProfileDetail>
             <ProfileImage>
-              <Avatar
-                size={58}
-                imgSrc={
-                  'https://firebasestorage.googleapis.com/v0/b/teamcamp-app.appspot.com/o/UserProfiles%2FUntitled1_1701236653470.jpg?alt=media&token=8bc07cdb-5fcc-4c69-8e0d-c9978b94b3e4'
-                }
-                name={''}
+              <input
+                type='file'
+                onChange={handleFileRead}
+                ref={inputRef}
+                style={{ display: 'none' }}
               />
+              {!isEmpty(image?.profile || user?.profile_url) ? (
+                <Avatar
+                  size={58}
+                  imgSrc={image?.profile ? image?.profile : user?.profile_url}
+                  name={''}
```

Comment:
name={''}

Use here user's name and only use one Avatar and remove another used same for unknown